jaxrs/src/main/java/org/killbill/billing/jaxrs/json/SubscriptionJson.java

```diff
@@ -87,6 +88,7 @@ public static class EventSubscriptionJson extends JsonBase {
         private final UUID eventId;
         private final BillingPeriod billingPeriod;
         private final DateTime effectiveDate;
+        private final Date catalogEffectiveDate;
```

Comment:
Can you use `DateTime` instead to be consistent, like [here](https://github.com/killbill/killbill/blob/work-for-release-0.23.x/jaxrs/src/main/java/org/killbill/billing/jaxrs/json/InvoiceItemJson.java#L68)

Reply:
HIi @sbrossie 
For catalog effectiveDate return type is Date as per  returned by "plan.getCatalog().getEffectiveDate() "

Reply:
@KBbitsP Please don;'t resolve comments until we agree on the resolution. I know `plan.getCatalog().getEffectiveDate()` returns a `Date` but all our json  models use `DateTime` so for consistency we need to convert it.

Reply:
@sbrossie Converted to DateTime. Please check